content/articles/ansible-and-nix.md

```diff
@@ -0,0 +1,333 @@
+Title: My experience of using NixOps as an Ansible user
+Date: 2015-05-12
+Short_summary: I reflect on my experience of having used Nix(OS/Ops) given my previous experience of Ansible
+Category: Infrastructure
+Authors: Gautier
+
+
+One of our client recently asked us to host and support their existing Django
+application.
+Although my usual tools for this kind of jobs are Ubuntu and
+[Ansible](http://www.ansible.com), I wanted to try the [Nix
+tools](https://nixos.org/nix/) in a real project. Tom has been praising their
+qualities since we started our company!
+<!-- PELICAN_END_SUMMARY -->
+
+In this article I am writing about my experience of using NixOps to deploy
+our client application on our infrastructure.
+
+I am illustrating the practical differences between NixOps and Ansible using a
+test project similar in terms of components to the application we deployed.
+
+But first, let me introduce Nix and Ansible.
+
+## Ansible
+
+My favourite configuration management tool for the last couple of years has
+been Ansible. It supports server configuration, server orchestration and
+even resources provisioning (like an S3 bucket or a Google Compute Engine
```

Comment:
Annette thinks "an S3 bucket" is better as this is how you would pronounce it and I've find more google results for "an S3 bucket" (About 50,400 results (0.29 seconds)) than "a S3 bucket" (About 16,100 results (0.25 seconds)).